client/apps/edit/components/content/sections/social_embed/test/controls.test.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import configureStore from 'redux-mock-store'
+import { Provider } from 'react-redux'
+import { cloneDeep } from 'lodash'
+import { mount } from 'enzyme'
+import { Fixtures } from '@artsy/reaction/dist/Components/Publishing'
+import { SectionControls } from '../../../section_controls'
+import { SocialEmbedControls } from '../controls'
+
+describe('EmbedControls', () => {
```

Comment:
All copied from `Embed`'s tests; will update them to be more specific to `SocialEmbed`

Reply:
👍 The height/mobile height ones can be removed too